api/tests/core/users/test_api.py

```diff
@@ -520,7 +522,7 @@ def test_with_eligible_underage_user(self):
             resultContent=fraud_factories.EduconnectContentFactory(registration_datetime=datetime.datetime.utcnow()),
         )
         user = subscription_api.activate_beneficiary_for_eligibility(
-            user, "test", users_models.EligibilityType.UNDERAGE
+            user, fraud_models.BeneficiaryFraudCheck(), users_models.EligibilityType.UNDERAGE
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏fraud_factories.BeneficiaryFraudCheckFactory()` serait plus cohérent pour les tests
(même si ça revienbt au même ici je crois)

Reply:
Edit: en fait c'est important, ça fait casser tes tests